<launch>
        
    <!-- "/hsrb/hand_camera/image_raw" -->
    <!-- "/hsrb/head_r_stereo_camera/image_raw" -->
    <node pkg="create_dataset" type="save_images" name="image_taker" output="screen">
        <remap 
            from="hsrb/head_center_camera/image_raw" 
            to="/camera/rgb/image_color"        
        />
    </node>
    

</launch>